packages/application-generic/src/logging/index.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  LoggerErrorInterceptor,
+  Logger,
+  LoggerModule,
+  PinoLogger,
+} from 'nestjs-pino';
+import { storage, Store } from 'nestjs-pino/storage';
+import * as process from 'process';
```

Comment:
This import line should be removed, process is available to the service and no need to import it.